site/src/contexts/auth/permissions.tsx

```diff
@@ -131,6 +132,12 @@ export const permissionsToCheck = {
     },
     action: "read",
   },
+  [checks.assignOrgRole]: {
+    object: {
+      resource_type: "assign_org_role",
+    },
+    action: "create",
+  },
```

Comment:
I'm worried about this check. it doesn't specify a specific org or `any_org`, so I'm not sure what response to expect here, and it looks like the check that you added to the `organizationPermissions` query is correct, and is the permission value that I think we should actually be using for all of this.

cc @Emyrk

Reply:
Yea this does not seem right. We should remove this imo, I see the other check as well

Reply:
Got it, I originally was trying to get custom roles working independently from organizations